minecraft-quilt/minecraft-quilt.json

```diff
@@ -0,0 +1,115 @@
+{
+  "type": "minecraft-java",
+  "display": "Quilt - Minecraft",
+  "data": {
+    "eula": {
+      "type": "boolean",
+      "desc": "Do you (or the server owner) agree to the <a href='https://account.mojang.com/documents/minecraft_eula'>Minecraft EULA?</a>",
+      "display": "EULA Agreement (true/false)",
+      "required": true,
+      "value": "false"
+    },
+    "game-version": {
+      "type": "string",
+      "desc": "Version of Minecraft to install (Oldest quilt version is 1.14)",
+      "display": "GAME VERSION",
+      "required": true,
+      "value": "1.19.3"
+    },
+    "ip": {
+      "type": "string",
+      "desc": "What IP to bind server to",
+      "display": "IP",
+      "required": true,
+      "value": "0.0.0.0"
+    },
+    "java": {

```

Comment:
As of v2.6, we use the javadl option to handle the java requirements.
See https://github.com/PufferPanel/templates/blob/v2.6/minecraft-vanilla/minecraft-vanilla.json#L52-L55 and https://github.com/PufferPanel/templates/blob/v2.6/minecraft-vanilla/minecraft-vanilla.json#L73 for example usages.